src/pages/homeView/Home.tsx

```diff
@@ -1,70 +1,73 @@
-import 'semantic-ui-css/semantic.min.css'
+import "semantic-ui-css/semantic.min.css";
 import React, { useState } from "react";
-import { Button } from 'semantic-ui-react'
+import { Button } from "semantic-ui-react";
 import "./Home.css";
-import teamImage from "../../images/team-image.svg"
-import Navbar from '../../components/navBar/Navbar';
-
+import teamImage from "../../images/team-image.svg";
+import Navbar from "../../components/navBar/Navbar";
 // import React, { useState } from "react";
```

Comment:
Same as above with the Navbar import, https://github.com/aryangupta1/SES2A-Group4-Frontend/pull/14/files/df24ea3825dae3dc09920e1350536b9417c5fa3c#r697940839